pkgs/top-level/all-packages.nix

```diff
@@ -37597,6 +37597,8 @@ with pkgs;
   foomatic-db-ppds = callPackage ../misc/cups/drivers/foomatic-db-ppds {};
   foomatic-db-ppds-withNonfreeDb = callPackage ../misc/cups/drivers/foomatic-db-ppds { withNonfreeDb = true; };
 
+  game-rs = callPackage ../misc/game-rs { };
```

Comment:
Please move the package to `pkgs/by-name/ga/game-rs/package.nix` and drop this line.

Reply:
sure, doing it rn

Reply:
the branch i have checked out doesn't have by-name folder, i'll rebase and push, sorry for the delay

Reply:
is it possible to keep it here and i move it to packages in next update? rebasing over so many commits is causing issues, i could open another PR and move the changes to a new branch which is based on a newer commit with by-name folder

Reply:
I'll do the rebase for you.

Reply:
thankyou so much😭 sorry for the inconvenience